UnityClient/Assets/Resources/Shaders/Sprites/BillboardSpriteShader.shader

```diff
@@ -0,0 +1,101 @@
+Shader "UnityRO/BillboardSpriteShader"

```

Comment:
Why isn't this billboard shader not using the commonly used Billboard.cginc?

Reply:
Did not know about a commonly used Billboard.cginc, We could check what type of rotation towards camera it does, probably something similar